sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -105,9 +105,9 @@ static void usb_send_str(const char *string, usb_ss_ctx_t *ss_ctx) {
 }
 
 // These GPIO bits control USB PHY configuration
-static const uint32_t kPinflipMask = 1;
-static const uint32_t kDiffXcvrMask = 2;
-static const uint32_t kUPhyMask = 4;
+static const uint32_t kPinflipMask = (1 << 8);
+static const uint32_t kDiffXcvrMask = (1 << 9);
+static const uint32_t kUPhyMask = (1 << 10);
```

Comment:
This is just a temporary fix to get the example functioning again on the CW310. This will get another pass after more alignment work on the CW310 pin-out.